### PR TITLE
make composeAll support custom loading and error

### PR DESCRIPTION
**Pull request by feit**

### CODE REVIEW

**Comment by arunoda** [comment]
Yeah! This is a huge bug.
Could you do this for `composeWithObservable` and `composeWithPromise` as well.

**Comment by feit** [comment]
supported, please review

File: lib/index.js [hunk 1]
```diff
@@ -200,12 +200,12 @@ export function composeWithObservable(fn) {
...
   };
 
-  return compose(onPropsChange);
+  return compose(onPropsChange, L, E);
 }
 
 // utility function to compose multiple composers at once.
 export function composeAll(...composers) {
-  return function (BaseComponent) {
+  return function (BaseComponent, L, E) {
```
Comment (arunoda) [inline lib/index.js]: We don't need it for here.

**Comment by feit** [comment]
updated

**Comment by arunoda** [comment]
Super. I'll take this.

**Comment by feit** [comment]
please bump a new version and publish to npm :)

**Comment by arunoda** [comment]
Done: v1.2.0

**Pull request merged**